README.MD

```diff
@@ -0,0 +1,22 @@
+# Virtual Baby Groot Pet
```

Comment:
Quite like the humour in this :) 

You do have multiple full size headings though (the ones with a single # at the beginning). There should only be one of these and it should be the title of the project. Heres a helpful cheatsheet [guide to markdown](https://www.markdownguide.org/cheat-sheet/)

Some installation and usage instructions would be a big plus here!